xbmc/guilib/GUIControlFactory.cpp

```diff
@@ -963,8 +961,18 @@ CGUIControl* CGUIControlFactory::Create(int parentID, const CRect &rect, TiXmlEl
   {
     StringUtils::ToLower(strTmp);
     if (strTmp == "horizontal")
+    {
       orientation = HORIZONTAL;
+      // default orientation for a settings slider is vertical
```

Comment:
It doesn't break horizontal sliders. If you look above the default orientation is `VERTICAL` [see](https://github.com/xbmc/xbmc/pull/10106/files#diff-d4fa1e4244e98591cf23ce6e5f167887R740).
Furthermore the values need to be changed otherwise the slider looks really ugly. It took very long to figure out this issue.